src/main/java/messages/OperationsMessages.java

```diff
@@ -0,0 +1,15 @@
+package messages;
+
+public class OperationsMessages {
+    public static final String INITIAL_MSG = "    Hello! I'm %s\n    What can I do for you?"; 
+    public static final String BYE_MSG = "     Bye. Hope to see you again soon!";
+    public static final String MARK_MSG = "     Nice! I've marked this task as done";
+    public static final String UNMARK_MSG = "     OK, I've marked this task as not done yet:";
+    public static final String SHOW_ITEMS_MSG = "    Here are the tasks in your list:";
+    public static final String ADDED_MSG = "    Got it. I've added this task:\n      %s\n    Now you have %d tasks in your list.";
```

Comment:
Java coding standard: Line length should not be longer than 120 chars.

Consider splitting it up into two lines using line wrapping.